docs/config.toml

```
@@ -70,11 +70,17 @@ pygmentsStyle = "friendly"
       identifier = "quickstart-credentials"
       weight = 3
       parent = "quickstart"
+    [[menu.main]]
+      name = "Configuration"
+      url = "/quickstart/configurations/"
```

It should be configuration (singular) as configurations isn't the right grammar. Please update the filename as well to match.

```suggestion
      url = "/quickstart/configuration/"
```